### PR TITLE
Implemented CassandraConnector.executeQuery

### PR DESCRIPTION
**Pull request by jcjimenez**
This is what I've been using for testing the other changes for the cassandra-based Settings resolvers like https://github.com/CatalystCode/project-fortis-services/pull/42

### CODE REVIEW

**Review by Smarker — APPROVED** [review]
LGTM

File: src/clients/cassandra/CassandraConnector.js [hunk 1]
```diff
@@ -19,7 +19,14 @@ function executeBatchMutations(mutations) { // eslint-disable-line no-unused-var
  */
 function executeQuery(query, params) { // eslint-disable-line no-unused-vars
   return new Promise((resolve, reject) => { // eslint-disable-line no-unused-vars
-
+    client.execute(query, params, {prepare:true}, (err, result)=>{
```
Comment (c-w) [inline src/clients/cassandra/CassandraConnector.js]: Nit: spacing consistency `prepare: true`, `(err, result) => {`.

**Review by c-w — APPROVED** [review]
LGTM.

**Review by Smarker — APPROVED** [review]
LGTM, I merged #28 so the part about Cassandra client is already on master now.

**Comment by jcjimenez** [comment]
Thanks, @Smarker

**Pull request merged**